ui2/src/common/Card.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { Card } from 'antd';
+import './Card.css';
+
+export const AppCard = ({ children, className, ...props }) => (
```

Comment:
looks a little bit redundant

Reply:
I have a css file that is crucial, and I thought it may be best to wrap the antd to the best of my ability, in case we want to switch the styling.